src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -40,6 +42,16 @@ async fn run() {
             return;
         }
 
+        // "encoded-userinfo-and-db.json" specifies a database name with a question mark which is
```

Comment:
Wow, I'm kind of surprised that's allowed at all

Reply:
The test seems to specifically be about having a URL-encoded user and database. Maybe it can use a different character that isn't prohibited anywhere, though.